### PR TITLE
v1.0

### PR DESCRIPTION
**Pull request by bpepple**
This PR is to finalize the v1.0 of the schema:

- Move MetronInfo.xsd & Sample.xml to schema directory
- Update tests
- Update README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,19 +2,56 @@
...
-Since Digital Comics are just are archive files (like .zip) this new XML schema can co-exist with any existing `ComicInfo.xml` if needed for backward compatibility.
+Since Digital Comics are just are archive files (like .zip) this new XML schema can co-exist with any existing
+`ComicInfo.xml` if needed for backward compatibility.
+
+## Is the schema only for the Metron Database?
+
+No, the schema only has *Metron* in the name since almost every other format has *Comic* in the name, and the
+originating author hates naming projects, so he went with the simplest choice. 😄 It was designed to be used for any of
+the
+comic resources (Comic Vine, AniList, etc.)
```
Comment (Buried-In-Code) [inline README.md:27]: Little nitpick, there's no need to have `the` wrapped on it's own line

**Comment by Buried-In-Code** [comment]
For the sample doc, I think this line should be updated to be a direct link to the schema, so something like this:

```diff
- <MetronInfo xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="MetronInfo.xsd">
+ <MetronInfo xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="https://raw.githubusercontent.com/Metron-Project/metroninfo/master/schema/v1.0/MetronInfo.xsd">
```

**Comment by bpepple** [comment]
> ```diff
> https://raw.githubusercontent.com/Metron-Project/metroninfo/master/schema/v1.0/MetronInfo.xsd
> ```

The line: xsi:noNamespaceSchemaLocation="MetronInfo.xsd" specifies **where** the schema resides (here it is in the same folder as "Sample.xml"), and if your IDE supports it (like PyCharm, VSCode, etc) it will validate it live. Changing it to what you suggested would most likely give you some type of error saying the uri is invalid.

**Comment by majora2007** [comment]
Description looks good to me. I'll leave the technical validation to you.

**Pull request merged**